func-tests/main.tf

```diff
@@ -183,27 +183,27 @@ resource "azion_edge_function" "testfunc2firewall" {
   }
 }
 
-resource "null_resource" "update_edge_functions" {
-  depends_on = [azion_edge_application_main_setting.testfunc]
-
-  provisioner "local-exec" {
-    command = "sleep 20 && terraform apply -auto-approve -target='azion_edge_application_main_setting.testfunc' -var 'edge_functions_module=true'"
-  }
-}
-
-resource "azion_edge_application_edge_functions_instance" "testfunc" {
-  depends_on = [null_resource.update_edge_functions]
+# resource "null_resource" "update_edge_functions" {
+#   depends_on = [azion_edge_application_main_setting.testfunc]
```

Comment:
don't we need these commented code sections?

Reply:
Currently I want to leave these comments to understand if we can uncomment this code and use it as an example for new tests